pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -61,6 +61,11 @@ const (
 	max_rand_index                = 25
 )
 
+const (
+	cmd_add = "ADD"
+	cmd_del = "DEL"
+)
```

Comment:
I don't think this is necessary (in general I don't see a point in defining constants for a short hardcoded string that's never going to change).

On a side note (but not important if you remove those), we don't use snake case for names, only camel case.